MapboxNavigation/PollyVoiceController.swift

```diff
@@ -181,12 +184,12 @@ public class PollyVoiceController: RouteVoiceController {
     
     func handle(_ awsTask: AWSTask<NSURL>) {
         guard awsTask.error == nil else {
-            callSuperSpeak(fallbackText, error: awsTask.error!.localizedDescription)
+            callSuperSpeak(fallbackText, error: awsTask.error!)
             return
         }
         
         guard let url = awsTask.result else {
-            callSuperSpeak(fallbackText, error: "No polly response")
+            callSuperSpeak(fallbackText, error: NSError.withCustom(message: "No AWS Task"))
```

Comment:
`localizedFailureReason` is meant to be human-readable and understandable by the end user, in case the developer presents the error via an alert box (not required). The best practice would be to make the NSError’s error code something generic, like `spokenInstructionFailed`, and its localized failure reason also vague, like “Unable to read aloud the instruction ‘…’.” Then we can add another field to the NSError’s user info dictionary that indicates exactly what went wrong (in this case, an AWS failure).